### ISSUE TITLE
Crash report 9.5: NPE in UsernameChangerFullScreenDialogFragment

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#7492]
```
Fatal Exception: java.lang.NullPointerException: Attempt to read from field 'java.util.List org.wordpress.android.ui.accounts.signup.UsernameChangerRecyclerViewAdapter.mItems' on a null object reference
       at org.wordpress.android.ui.accounts.signup.UsernameChangerFullScreenDialogFragment.onConfirmClicked(UsernameChangerFullScreenDialogFragment.java:190)
       at org.wordpress.android.ui.FullScreenDialogFragment.onConfirmClicked(FullScreenDialogFragment.java:242)
       at org.wordpress.android.ui.FullScreenDialogFragment$4.onMenuItemClick(FullScreenDialogFragment.java:225)
       at android.support.v7.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:167)
       at android.support.v7.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:973)
       at android.support.v7.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:963)
       at android.support.v7.widget.ActionMenuView.invokeItem(ActionMenuView.java:624)
       at android.support.v7.view.menu.ActionMenuItemView.onClick(ActionMenuItemView.java:150)
       at android.view.View.performClick(View.java:6891)
       at android.widget.TextView.performClick(TextView.java:12651)
       at android.view.View$PerformClick.run(View.java:26083)
       at android.os.Handler.handleCallback(Handler.java:789)
       at android.os.Handler.dispatchMessage(Handler.java:98)
       at android.os.Looper.loop(Looper.java:164)
       at android.app.ActivityThread.main(ActivityThread.java:6938)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374)

```

**Comment by maxme** [comment]
http://crashes.to/s/b3bdb4b8e17

**Comment by mzorz** [comment]
closed via #7494
